test/mocha/helpers.js

```diff
@@ -171,17 +171,30 @@ async function insertTestData(mockData) {
   }
 }
 
-api.signDocument = async ({doc, verificationMethod, key}) => {
+api.signDocument = async ({doc, invocationTarget, key}) => {
+  const contextArray = Array.isArray(doc['@context']);
+  // if the context is an array add the ZCAP context if it's not there
```

Comment:
```suggestion
  // if the context is an array and the ZCAP context isn't included, add it
```

Reply:
this might add a linting error for line length. The next line continues the sentence.